tests/testthat/test-verify-output.R

```diff
@@ -0,0 +1,27 @@
+test_that("can record all types of output", {
+  verify_output(test_path("test-verify-output.txt"), {
+    "Output"
+    1 + 2
+    invisible(1:10)
+    12345678 + 12345678 + 12345678 + 12345678 + 12345678 + 12345678 +
```

Comment:
Setting the width option must need to be moved slightly, as with a larger width option this output is different.